examples/conf/hekad.toml

```diff
@@ -0,0 +1,9 @@
+[hekad]
+maxprocs = 2
+
+[TcpInput]
+address = "127.0.0.1:5565"
+parser_type = "message.proto"
+decoder = "ProtobufDecoder"
```

Comment:
`parser_type` is no longer a supported config option, it's been replaced by splitter plugins. If you just remove the `parser_type` and `decoder` settings entirely, Heka will choose the right defaults.